routes/users/users.js

```diff
@@ -0,0 +1,30 @@
+const users = require("express").Router();
+
+let peopleArr = [{
+    username: "user1234",
+    email: "user1234@gmail.com",
+    activationDate: "January 1, 1970"
+}]
+
+users.get("/", (req, res) => {
+    res.json(peopleArr)
+})
+
+users.get("/:id", (req, res) => {
+    res.json(res.json(peopleArr[req.params.id]))
```

Comment:
You dont need to nest res.json, using it once is fine. `res.json(peopleArr[req.params.id])`